### ISSUE TITLE
Potential issue and fix

### ISSUE THREAD

**Issue by POE-Addon-Launcher** [POE-Addon-Launcher/PAL2#50]
https://www.reddit.com/r/pathofexile/comments/bwezvu/release_poe_addon_launcher_2_v111_start_legion_up/er4cizp/?context=3

**Comment by ghost** [comment]
yeah, this is totally because of the filter checking. my install would hang during boot about 9 times out of 10 until I removed filters from pal, now it boots every time. it's definitely failing to connect to filterblade or whatever and hanging

**Comment by POE-Addon-Launcher** [comment]
Yeah the filterblast API is pretty annoying at times, though I thought I made the program run even when filterblast isn't working.

**Comment by ghost** [comment]
The exact same thing happens in my case.
Also, the PAL_Logger.log seems to have the wrong format for the date. It should be "year-month-day" and afterwards the time, right? But the month is replaced with minutes. So it'll show something like "2019-51-03".
